Bio/PDB/parse_pdb_header.py

```diff
@@ -219,6 +220,9 @@ def _parse_pdb_header_list(header):
             rr = re.search(r"\d\d-\w\w\w-\d\d", tail)
             if rr is not None:
                 dict['deposition_date'] = _format_date(_nice_case(rr.group()))
+            rr = re.search(r"\s+\w\w\w\w\s*\Z", tail)
```

Comment:
The first character of a PDB ID is always digit 1-9:

http://www.wwpdb.org/documentation/file-format-content/format33/sect1.html#Field

Also, `\w` matches _, which isn't a valid character in a PDB ID.

I'd suggest `r"\s+([1-9][0-9A-Z]{3})\s*\Z"` and using `group(1)` to exclude the whitespace.

Reply:
Yes definitely, thank you for both suggestions.  Also updated NEWS and CONTRIB as suggested.

Reply:
Thanks for updating the regex. It would also be nice to add one line to an existing unittest in `test_PDB_parse_pdb_header.py`.

Reply:
Done.  Thank you for getting me to learn about the testing framework :-), I think implementation I've done is correct..